testing/build.gradle

```diff
@@ -18,7 +18,7 @@ ext {
     noPublish = true
 }
 dependencies {
-    implementation project(':processor')
+    implementation testFixtures(project(':processor'))
```

Comment:
Even after reading https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures this I still don't get why is this necessary... can you explain what's it for?

Reply:
I wanted `TestTracingProvider`/`TestTraceHandle` to be used in both `processor` tests and in `testing`, but not to be part of `processor`'s main classpath. So test fixtures seemed like the right place to put them.

Reply:
hm, I see https://stackoverflow.com/questions/5644011/multi-project-test-dependencies-with-gradle/60138176#60138176